test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -307,6 +307,7 @@ public void UseBasePathConfiguresBasePath()
         [Fact]
         public void RelativeApplicationBaseAreResolved()
         {
+            // Arrange & Act
```

Comment:
Just a habit from MVC work I suppose, I'll remove them if that's not the convention for these repos.